lib/github_service.rb

```diff
@@ -11,12 +11,22 @@ module GithubService
   # may already be well handled by Octokit itself.
   #
   class << self
-    def add_comments(fq_repo_name, issue_number, comments)
+    def add_comments(fq_repo_name, issue_number, comments, commit_status = nil)
       Array(comments).each do |comment|
-        add_comment(fq_repo_name, issue_number, comment)
+        retval = add_comment(fq_repo_name, issue_number, comment)
+
+        unless commit_status.nil?
+          commit_status["options"]["target_url"] = retval["html_url"]
+          add_status(commit_status)
```

Comment:
Adding the status is depending on the comment's ID, however, it is an operation on the same level so this method call doesn't belong here, but AFTER the `add_comments` is called.